db/migrate/20180924185921_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,58 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[5.2]
+  def change
+
+    #drop_table :users
```

Comment:
It's better to have code without useless comments.

Reply:
Same for all useless comments on this file.

Reply:
Deleted all the useless comments.